src/core/src/OmniTileset.cpp

```diff
@@ -458,6 +458,7 @@ void OmniTileset::reload() {
     options.culledScreenSpaceError = getCulledScreenSpaceError();
     options.mainThreadLoadingTimeLimit = getMainThreadLoadingTimeLimit();
     options.showCreditsOnScreen = getShowCreditsOnScreen();
+    options.excluders = std::vector<std::shared_ptr<Cesium3DTilesSelection::ITileExcluder>>();
```

Comment:
No need to manually initialize the vector. This line can be removed.

Reply:
Interesting. In C#, you have to initialize their version of `std::vector`, but it's slightly different due to the `new`.